emanote/src/Emanote/Model/Note.hs

```diff
@@ -406,4 +406,10 @@ applyNoteMetaFilters doc =
               pure $ SData.oneAesonText (toList key) val
           )
 
+-- TODO: apply this recursively
```

Comment:
You can address this TODO using [`Text.Pandoc.Walk.walk`](https://hackage.haskell.org/package/pandoc-types-1.23.1/docs/Text-Pandoc-Walk.html#v:walk).